src/app/shared/services/translation.service.ts

```diff
@@ -41,6 +41,7 @@ constructor(
             this.setIndex();
         } else {
             const lang = HindiLang;
+            lang.sort(this.sortLang);
```

Comment:
@Harshwardhan18 You didn't use the script for sorting?

Reply:
@Manzurkds I used the script but somehow the sorted JSON file was not working I tried this and it worked so added and removed the need to sort the script from an outer source.

Reply:
@Harshwardhan18 Don't add it here, create a sortTranslation.js file in the repo which will import the json file, sort the content, and then write the sorted content on the file.
Add a script tag in package.json to run this file like 
`sort-translations: node sortTranslations.js`
Run this script tag before doing a build

Reply:
Adding it here means, we will be sorting the translations again and again on every load, which doesn't make any sense.